databricks_cli/runs/cli.py

```diff
@@ -39,21 +41,51 @@
               help='File containing JSON request to POST to /api/2.*/jobs/runs/submit.')
 @click.option('--json', default=None, type=JsonClickType(),
               help=JsonClickType.help('/api/2.*/jobs/runs/submit'))
+@click.option('--wait', is_flag=True,
+              help='If specified, the CLI will wait for the submitted run to complete ' +
+                   '(only available in API 2.1).')
 @api_version_option
 @debug_option
 @profile_option
 @eat_exceptions
 @provide_api_client
-def submit_cli(api_client, json_file, json, version):
+def submit_cli(api_client, json_file, json, wait, version):
     """
     Submits a one-time run.
 
     The specification for the request json can be found
     https://docs.databricks.com/api/latest/jobs.html#runs-submit
     """
     check_version(api_client, version)
-    json_cli_base(json_file, json, lambda json: RunsApi(
-        api_client).submit_run(json, version=version))
+    api_version = version or api_client.jobs_api_version
+    if api_version != '2.1' and wait:
+        click.echo(click.style('ERROR', fg='red') + ': the option --wait is only available ' +
+                   'in API 2.1', err=True)
```

Comment:
Why?

Reply:
I was concerned that runs-get call will have a different response structure between `2.1` and `2.0`. However, I tested the `--wait` option on `2.0` and it still worked. So I will remove this check.